src/Compilers/CSharp/Portable/Declarations/MergedTypeDeclaration.cs

```diff
@@ -46,15 +46,19 @@ public ImmutableArray<SyntaxReference> SyntaxReferences
             }
         }
 
-        public ImmutableArray<SyntaxList<AttributeListSyntax>> GetAttributeDeclarations()
+        public ImmutableArray<SyntaxList<AttributeListSyntax>> GetAttributeDeclarations(QuickAttributes? quickAttributes)
```

Comment:
nit: xml doc? Those are the quick attributes flags for the attributes we're looking for

Reply:
Done.